README.md

```diff
@@ -583,22 +572,37 @@ In this case, you will need to configure the following additional settings that
 Nexus instance that hosts the permanent content: `cachito_nexus_hoster_username`,
 `cachito_nexus_hoster_password`, and `cachito_nexus_hoster_url`.
 
+On each request, Cachito will create a proxy repository to the group repository of the request's
+package manager (e.g. `cachito-pkg-manager`). Cachito will populate this proxy repository to
+contain the subset of dependencies declared in the repository's lock file. Once populated, Cachito
+will block the repository from getting additional content. This prevents the consumer of the
+repository from installing something that was not declared in the lock file. This is further
+enforced by locking down the repository to a single user created for the request, which the
+consumer will use. Please keep in mind that for this to function properly, anonymous access needs
+to be disabled on the Nexus instance or at least not set to have read access on all repositories.
+
+These repositories and users created per request are deleted when the request is marked as stale
+or the request fails.
```

Comment:
I don't think this should be in the general section, many of these points are unique to npm and yarn at the moment.